Add John Goure to Contributors list

JohnGoure opened the pull request.

JohnGoure committed:
Add John Goure to Contributors list

Roshanjossey commented:
Hi  @JohnGoure, I'm quite elated about your pull request. I wanna evolve this project to addresses various problems faced by first-time contributors. I'd love to learn about your journey in open source community, the problems, pain points you had etc.
Could you explain how you felt when you went through the tutorial, made a pull request and learned that I merged it?

 We’ve recently added social share  to our web app. Could you please go to https://roshanjossey.github.io/first-contributions/#social-share and share your first contribution to open source?  Also, check out projects with easy issues while you’re there.

JohnGoure commented:
Hi @Roshanjossey, I appreciate you creating this project. It was an enlightening experience. What kind of problems are users running into?

I would be glad to describe my experience to you. When I got to the part where I cloned the project I thought to myself, "If I wasn't familiar with CLI and GIT I would probably be frustrated at this point." I then went through the steps of adding my name to the contributors.md file. Then I checked the status as you said to do. Then I added the change. Then I committed it with a message. 

When I tried pushing the branch to my master I ran into the problem of authentication. So I googled how to change the user email config. I changed my email and it was smooth sailing.

Roshanjossey commented:
@JohnGoure, some of the problems around contributing for other users is the effort for finding how a project works

I’d urge you to continue contributing to other open source projects.
We also have a slack team to help/mentor people trying to contribute to open source projects.
You can join the team by going to https://goo.gl/PM1Jdh
